src/resources/model.js

```diff
@@ -62,6 +62,7 @@ pc.extend(pc, function () {
             var data = asset.data;
 
             resource.meshInstances.forEach(function (meshInstance, i) {
+                meshInstance.mesh.area = data.area;

```

Comment:
Yeah maybe. Because what happens with procedurally generated meshes? These won't have area calculated so if this is only server-side then it should probably stay in the asset only.